.github/workflows/backend-checks.yml

```diff
@@ -44,6 +44,7 @@ jobs:
 
   may-merge:
     needs: ['lint', 'workspace-dependencies']
+    if: ${{ always() }}
```

Comment:
Note: `always()` also works without the curly braces, however in [the docs](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/using-jobs-in-a-workflow#example-not-requiring-successful-dependent-jobs), curly braces are used, so I use the same form as the docs just to me sure.